contracts/external/proxies/LiquidatorProxyV1ForSoloMargin.sol

```diff
@@ -0,0 +1,446 @@
+/*
+
+    Copyright 2019 dYdX Trading Inc.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+*/
+
+pragma solidity 0.5.7;
+pragma experimental ABIEncoderV2;
+
+import { SafeMath } from "openzeppelin-solidity/contracts/math/SafeMath.sol";
+import { ReentrancyGuard } from "openzeppelin-solidity/contracts/utils/ReentrancyGuard.sol";
+import { SoloMargin } from "../../protocol/SoloMargin.sol";
+import { Account } from "../../protocol/lib/Account.sol";
+import { Actions } from "../../protocol/lib/Actions.sol";
+import { Decimal } from "../../protocol/lib/Decimal.sol";
+import { Math } from "../../protocol/lib/Math.sol";
+import { Monetary } from "../../protocol/lib/Monetary.sol";
+import { Require } from "../../protocol/lib/Require.sol";
+import { Types } from "../../protocol/lib/Types.sol";
+import { OnlySolo } from "../helpers/OnlySolo.sol";
+
+
+/**
+ * @title LiquidatorProxyV1ForSoloMargin
+ * @author dYdX
+ *
+ * Contract for liquidating other accounts in Solo. Does not take marginPremium into account.
+ */
+contract LiquidatorProxyV1ForSoloMargin is
+    OnlySolo,
```

Comment:
Why is this needed? I don't see the `onlySolo` modifier used anywhere

Reply:
Oh I see you're just using it for the `SOLO_MARGIN` state variable. I guess that's fine